### ISSUE TITLE
Documentation Code Segments Throw TANF-related error

### ISSUE THREAD

**Issue by yvvijay121** [PolicyEngine/openfisca-us#635]
Every `IndividualSim` that is created throws the following error. This is probably caused by #604, where we added `CA_1` to distinguish between TANF maximum amount payment levels in different regions of California.
```
WARNING:root:Parameter hhs.tanf.cash.amount.max has a child CA_1 that is not in the possible values of state_code, ignoring.
```
![image](https://user-images.githubusercontent.com/62124499/160529891-240abce1-cc28-46f8-ac59-f1e45c14efc6.png)

**Comment by MaxGhenis** [comment]
Thanks for patching this up by picking the first region within California (https://github.com/PolicyEngine/openfisca-us/pull/636). To model this fully, we'll need to add a `tanf_region` variable.

**Comment by yvvijay121** [comment]
Issue fixed with PR #636. I'll make a separate issue to create a `tanf_region` variable. Thanks!
